src/ng-select/lib/ng-select.module.ts

```diff
@@ -57,6 +57,15 @@ import { DefaultSelectionModelFactory } from './selection-model';
 		NgLoadingSpinnerTemplateDirective,
 		NgClearButtonTemplateDirective
 	],
-	providers: [{ provide: SELECTION_MODEL_FACTORY, useValue: DefaultSelectionModelFactory }],
+	providers: provideNgSelect(),
```

Comment:
Not a fan, every call to a function needs to be avoided if not having a practical function. Pun intended.